src/cmd/serial.js

```diff
@@ -148,10 +148,9 @@ module.exports = class SerialCommand {
 		const handleClose = () => {
 			if (follow && !cleaningUp){
 				console.log(chalk.bold.white('Serial connection closed.  Attempting to reconnect...'));
-				reconnect();
-			} else {
-				console.log(chalk.bold.white('Serial connection closed.'));
+				return reconnect();
 			}
+			console.log(chalk.bold.white('Serial connection closed.'));
```

Comment:
Minor suggestion: `'\nSerial connection closed.'` 
and : `'\nSerial connection closed.  Attempting to reconnect...'`

Changes:
```
0000528683 [app] INFO: count:00115260
0000528684 [app] INFO: count:00115261
0000528685 [app] INFO: count:00115262
0000528685 [app] INFSerial connection closed.
```
To:
```
0000608934 [app] INFO: count:00125414
0000608935 [app] INFO: count:00125415
0000608935 [app] INFO: count:
Serial connection closed.
```

but tested and working well!  Thanks for the quick turn around!!

Reply:
✅ in https://github.com/particle-iot/particle-cli/pull/574/commits/0a8d581194cce4bfc70fef39a6d310b71b2f5665

nice catch 👍